SDKAndroid/src/main/res/layout/activity_kycactivity.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Think `activity_kyc` is a better name. It's redundant as-is.

Reply:
Done!